target/linux/ath79/dts/qca9531_8dev_lima.dts

```diff
@@ -80,6 +80,10 @@
 					macaddr_art_6: macaddr@6 {
 						reg = <0x6 0x6>;
 					};
+
+					calibration_art_1000: calibration@1000 {
+						reg = <0x1000 0x800>;
```

Comment:
All SoC integrated ath9k wireless cal data sizes are `0x440`. Include AR933x, AR934x and QCA95xx series.

Reply:
Larger EEPROMs can also be accepted by the ath9k driver. I think the script is indeed incorrect. It doesn't make sense to use different EEPROM sizes for the same chip and the same driver.

Reply:
@andreybondar59 can you verify this?

Reply:
@pepe2k might be able to as well.

Reply:
looking at this again, I think you're right. 

I'll rework this PR and push later.